torch/package/package_exporter.py

```diff
@@ -382,11 +383,7 @@ def require_module_if_not_provided(self, module_name: str, dependencies=True):
             return
 
         if self._can_implicitly_extern(module_name):
-            if self.verbose:
-                print(
-                    f"implicitly adding {module_name} to external modules "
-                    f"since it is part of the standard library and is a dependency."
-                )
+            self._implicitly_externed.append(module_name)
```

Comment:
Since `DiGraph` allows you to add arbitrary node attributes, consider record the fact that an extern is implicit in the graph node itself and so you can avoid introducing this new attribute.